lib/src/model/broadcast/broadcast.dart

```diff
@@ -4,46 +4,45 @@ import 'package:freezed_annotation/freezed_annotation.dart';
 
 part 'broadcast.freezed.dart';
 
+@freezed
+class BroadcastResponse with _$BroadcastResponse {
+  const factory BroadcastResponse({
+    required IList<Broadcast> active,
+    required IList<Broadcast> upcoming,
+    required IList<Broadcast> past,
+  }) = _BroadcastResponse;
+}
+
 @freezed
 class Broadcast with _$Broadcast {
   const Broadcast._();
 
   const factory Broadcast({
     required BroadcastTournament tour,
-    required IList<BroadcastRound> rounds,
+    required BroadcastRound lastRound,
+    required String? group,
   }) = _Broadcast;
 
-  BroadcastRound? get curentRound =>
-      rounds.where((r) => r.status == RoundStatus.live).firstOrNull ??
-      rounds.where((r) => r.status == RoundStatus.finished).lastOrNull;
+  factory Broadcast.loading() => Broadcast(
+        tour: const BroadcastTournament(name: '', imageUrl: null),
+        lastRound: BroadcastRound(
+          id: '',
```

Comment:
I missed that in the last PR, but we should avoid `String` for ids and use an extension type instead. See `common/id.dart` for example.

Reply:
Should I create a BroadcastRoundId ?